<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Items">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Skills">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Skill3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 68px;
background-color: red;
height: 68px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 125)
</string>
    </property>
    <property name="text">
     <string>Skill3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Skill5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 68px;
background-color: red;
height: 68px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 125)
</string>
    </property>
    <property name="text">
     <string>Skill5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Hero1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white ;
border: 1px solid rgba(255, 255, 255, 125)</string>
    </property>
    <property name="text">
     <string>Pos 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Talant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 68px;
background-color: red;
height: 68px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 100)
</string>
    </property>
    <property name="text">
     <string>Talents</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Hero3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white ;
border: 1px solid rgba(255, 255, 255, 125)</string>
    </property>
    <property name="text">
     <string>Pos 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Skill4Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>272</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Picture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>146</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(heroes/images/DarkWillow.png);
width: 255px;
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>971</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Hero2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 125)
</string>
    </property>
    <property name="text">
     <string>Pos 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Skill2Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>250</y>
      <width>232</width>
      <height>290</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white ;
</string>
    </property>
    <property name="text">
     <string>Skywrath Mage(Петух)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Lore">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>971</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Skill3Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>250</width>
      <height>284</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Hero5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white ;
border: 1px solid rgba(255, 255, 255, 125)</string>
    </property>
    <property name="text">
     <string>Pos 5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Hero4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white ;
border: 1px solid rgba(255, 255, 255, 125)</string>
    </property>
    <property name="text">
     <string>Pos 4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Skill4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 68px;
background-color: red;
height: 68px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 125)
</string>
    </property>
    <property name="text">
     <string>Skill4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Skill5Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>250</y>
      <width>300</width>
      <height>229</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Skill2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 68px;
background-color: red;
height: 68px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 125)
</string>
    </property>
    <property name="text">
     <string>Skill2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Skill1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 68px;
background-color: red;
height: 68px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 125)
</string>
    </property>
    <property name="text">
     <string>Skill1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Skill1Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>250</y>
      <width>210</width>
      <height>284</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TalentsInfo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>280</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>Items</zorder>
   <zorder>Skills</zorder>
   <zorder>Skill3</zorder>
   <zorder>Skill5</zorder>
   <zorder>Hero1</zorder>
   <zorder>Talant</zorder>
   <zorder>Hero3</zorder>
   <zorder>Picture</zorder>
   <zorder>label_9</zorder>
   <zorder>Hero2</zorder>
   <zorder>Name</zorder>
   <zorder>Lore</zorder>
   <zorder>Hero5</zorder>
   <zorder>Hero4</zorder>
   <zorder>Skill4</zorder>
   <zorder>Skill2</zorder>
   <zorder>Skill1</zorder>
   <zorder>TalentsInfo</zorder>
   <zorder>Skill3Info</zorder>
   <zorder>Skill1Info</zorder>
   <zorder>Skill5Info</zorder>
   <zorder>Skill4Info</zorder>
   <zorder>Skill2Info</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
